acceuil.md

```diff
@@ -0,0 +1,282 @@
+---
+title: GCWeb, le thème de Canada.ca
+altLangPage: home.html
+dateModified: 2020-05-13
+description: "Page d'accueil décrivant l'ensemble des composants du thème de Canada.ca, nommé GCWeb."
+lang: fr
+---
+
+{::nomarkdown}
+<div class="row">
+	<div class="col-md-7 col-lg-8">
+		<p>Les gabarits et les conceptions communes si dessous sont une référence d'implémentation du <a href="https://conception.canada.ca">Système de conception de Canada.ca</a>, incluant les exigences obligatoire de la spécifications du contenu et de l’architecture de l'information (C&amp;AI) pour Canada.ca. Les ministères et organisme du gouvernement du Canada peuvent y contribuer en publiant leur modèle et leur conception commune via le <a href="https://github.com/wet-boew/GCWeb">dépôt github de GCWeb</a>.</p>
+	</div>
+	<div class="col-xs-12 col-md-auto pull-right">
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.3.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v9.3.0</strong></a><br />
```

Comment:
@duboisp We would need to find a way to have the version number and downloadable file to update automatically with every release.

Reply:
Yes, this would be feasible by using the data-json and the json manager with the GitHub Restful API.